packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -3978,6 +4054,26 @@ Specifies a hashed password to import into Okta. This allows an existing passwor
 }
 ```
 
+##### Password Hook Object
+
+Specifies to use a password hook to import user into Okta. This allows an existing password to be imported into Okta directly from some other store. See [Create User with Password Hook](#create-user-with-password-hook) for information on using this object when creating a user. When updating a user with a password hook the user must be in the `STAGED` status.
+
+> **Note:** Because the plain text password isn't specified when a password hook is specified, password policy isn't applied.
+
+| Property   | DataType | Description                                                                                                                                                                                | Required                                                                      | Min Value                      | Max Value                      |
+|:-----------|:---------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------|:-------------------------------|:-------------------------------|
+| type  | String   | The type of password hook. Must be set to default.                                                                                            | TRUE                                                                          | N/A                            | N/A                            |
+
```

Comment:
"password hook" > "Password Inline Hook" 
"Must be set to default." > Maybe, to give an indication about why this exists: " "Currently, must be set to default."

Reply:
Fixed!